src/js/components/Notification/Notification.js

```diff
@@ -0,0 +1,70 @@
+import React, { useContext } from 'react';
+import { ThemeContext } from 'styled-components';
+import { defaultProps } from '../../default-props';
+
+import { Layer } from '../Layer';
+import { Box } from '../Box';
+import { Button } from '../Button';
+import { Text } from '../Text';
+import { Paragraph } from '../Paragraph';
+
+const Notification = ({ toast, message, body, status, onClose }) => {
```

Comment:
change `message` --> `title` to align with notifications API,

potentially add `onClick`

Reply:
`body` --> `content`